seacrowd/sea_datasets/xquad/xquad.py

```diff
@@ -0,0 +1,157 @@
+            features = schemas.qa_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        name_split = self.config.name.split("_")
+        subset_name = name_split[0]
+
+        if subset_name == "xquad":
+            # For 'xquad' subset, download and combine Vietnamese and Thai subsets
+            vi_filepath = dl_manager.download_and_extract(_URLS + "xquad.vi.json")
+            th_filepath = dl_manager.download_and_extract(_URLS + "xquad.th.json")
+            filepaths = [vi_filepath, th_filepath]
+        else:
+            # For specific language subsets
+            filepath = dl_manager.download_and_extract(_URLS + subset_name + ".json")
+            filepaths = [filepath]
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepaths": filepaths,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepaths: List[Path], split: str) -> Tuple[int, Dict]:
+        count = 0
+        document_count = 0
+        for filepath in filepaths:
+            with open(filepath, encoding="utf-8") as f:
+                data = json.load(f)
+
+            for paragraphs in data["data"]:
+                for example in paragraphs["paragraphs"]:
+                    context = example["context"]
+                    for qa in example["qas"]:
+                        question = qa["question"]
+                        answers = qa["answers"]
+                        answers_start = [answer["answer_start"] for answer in answers]
+                        answers_text = [answer["text"] for answer in answers]
+
+                        if self.config.schema == "source":
+                            yield count, {
+                                "context": context,
+                                "question": question,
+                                "answers": {"answer_start": answers_start, "text": answers_text},
+                                "id": count,
+                            }
+                        elif self.config.schema == "seacrowd_qa":
+                            yield count, {
+                                "question_id": count,
+                                "context": context,
+                                "question": question,
+                                "answer": {"answer_start": answers_start[0], "text": answers_text[0]},
+                                "id": count,
+                                "choices": [],
+                                "type": "extractive",
+                                "document_id": document_count,
+                            }
```

Comment:
Add meta field from new SEACrowd schema for QA
```suggestion
                                "document_id": document_count,
                                "meta": {}
                            }
```